fibonacci.ts

```diff
@@ -12,7 +12,7 @@
 
 const fibonacciValue = (number: number): number => {
   if (number < 1) {
-    return 0; // If given number is less than 1 than return 0
+    return 0; 
```

Comment:
Perhaps throw an error for negative or non-integer numbers, and return 0 for `number === 0`?